chsdi/models/vector/stopo.py

```diff
@@ -3097,20 +3097,20 @@ class AmtlichesAdressVerzeichnis(Base, Vector):
     __table_args__ = ({'schema': 'vd', 'autoload': False})
     __template__ = 'templates/htmlpopup/addressverzeichnis.mako'
     __bodId__ = 'ch.swisstopo.amtliches-gebaeudeadressverzeichnis'
-    __label__ = 'id'
-    __queryable_attributes__ = ['id', 'bdg_egid', 'adr_edid', 'str_esid', 'str_label', 'adr_number']
-    id = Column('adr_egaid', BigInteger, primary_key=True)
+    __label__ = 'adr_egaid'
+    __queryable_attributes__ = ['adr_egaid', 'bdg_egid', 'adr_edid', 'str_esid', 'stn_label', 'adr_number']
+    id = Column('id', BigInteger, primary_key=True)
```

Comment:
is it not possible to just referene adr_egaid as source here? Why do we have to fill two columns with the same value?

Reply:
@ltclm according to the ticket we want to return `adr_egaid` as attribute name. if my test were right it is not possible to reference the same column to different attributes in the model, hence the duplication.

Reply:
ok, thank you!